blocks/api/registration.js

```diff
@@ -80,10 +80,14 @@ export function registerBlockType( name, settings ) {
 		return;
 	}
 	if ( blocks[ name ] ) {
-		console.error(
-			'Block "' + name + '" is already registered.'
-		);
-		return;
+		if ( module.hot ) {
+			delete( blocks[ name ] );
```

Comment:
`delete` is a keyword, not a function, and while I expect this should work as-is, the syntax could lead to confusion. I'd suggest:

```js
delete blocks[ name ];
```